task/clone-refs/0.1/README.md

```diff
@@ -0,0 +1,53 @@
+# Unit Tests for Go Code
+
+The clonerefs Task clone repositories references defined by prowjob refs and extra-refs.
```

Comment:
again, there's inconsistency in naming convention. The directory is called `clone-refs` but you refer it as `clonerefs` here